docs/sources/setup/install/helm/install-scalable/_index.md

```diff
@@ -49,72 +53,51 @@ It is not recommended to run scalable mode with `filesystem` storage. For the pu
 
 3. Create the configuration file `values.yaml`. The example below illustrates how to deploy Loki in test mode using MinIO as storage:
 
-     ```yaml
-     loki:
-       schemaConfig:
-         configs:
-           - from: 2024-04-01
-             store: tsdb
-             object_store: s3
-             schema: v13
-             index:
-               prefix: loki_index_
-               period: 24h
-       ingester:
-         chunk_encoding: snappy
-       tracing:
-         enabled: true
-       querier:
-         # Default is 4, if you have enough memory and CPU you can increase, reduce if OOMing
-         max_concurrent: 4
-
-     #gateway:
-     #  ingress:
-     #    enabled: true
-     #    hosts:
-     #      - host: FIXME
-     #        paths:
-     #          - path: /
-     #            pathType: Prefix
-
-     deploymentMode: SimpleScalable
-
-     backend:
-       replicas: 3
-     read:
-       replicas: 3
-     write:
-       replicas: 3
-
-     # Enable minio for storage
-     minio:
-       enabled: true
-
-     # Zero out replica counts of other deployment modes
-     singleBinary:
-       replicas: 0
-
-     ingester:
-       replicas: 0
-     querier:
-       replicas: 0
-     queryFrontend:
-       replicas: 0
-     queryScheduler:
-       replicas: 0
-     distributor:
-       replicas: 0
-     compactor:
-       replicas: 0
-     indexGateway:
-       replicas: 0
-     bloomCompactor:
-       replicas: 0
-     bloomGateway:
-       replicas: 0
-     ```
-
-4. Install or upgrade the Loki deployment.
+    ```yaml
+      loki:
+        schemaConfig:
+          configs:
+            - from: 2024-04-01
+              store: tsdb
+              object_store: s3
+              schema: v13
+              index:
+                prefix: loki_index_
+                period: 24h
+        ingester:
+          chunk_encoding: snappy
+        querier:
+          # Default is 4, if you have enough memory and CPU you can increase, reduce if OOMing
+          max_concurrent: 4
+        pattern_ingester:
+          enabled: true
+        limits_config:
+          allow_structured_metadata: true
+          volume_enabled: true
+          retention_period: 672h
+        compactor:
+          retention_enabled: true 
+          delete_request_store: s3
+
+      deploymentMode: SimpleScalable
+
+      backend:
+        replicas: 2
+      read:
+        replicas: 2
+      write:
```

Comment:
is it worth a comment that this is 3 because of the replication factor, while the others can be 2 because they don't deal with replication?

Reply:
I added a note within the yaml to explain. Great point and one I will highlight in the video